vllm/model_executor/layers/quantization/compressed_tensors/utils.py

```diff
@@ -214,3 +224,24 @@ def _match_fused_layer(
             unfused_matches.append(None)
 
     return unfused_matches[0] if all(unfused_matches) else None
+
+
+def fp8_channelwise_quantize(x: Tensor, channel_dim: int = -1) -> tuple[Tensor, Tensor]:
```

Comment:
is there a higher level function in `compressed-tensors` to do this so we don't have to redefine it here? @kylesayrs  since I'm very n00b in compressed-tensors